website/docs/basics/blocks-styles.md

```diff
@@ -330,155 +327,186 @@ If the `variable` key is not set in options, the `value` key will be used as fal
+### Responsive Inverse
+
+By default, we use mobile first approach but if you need desktop first you can use `inverse: true` key.
 
-If you need a more complex setup with multiple variables for a single attribute value you can use the `custom` key. It will output the CSS variables based on values defined in the `options`. Each custom variable key must be an `object`.
+> If you have multiple mobile/desktop first breakpoints, output will sort them mobile first and then desktop first after that.
 
 **Manifest:**
 ```json
 {
+	"componentClass": "typography",
 	"attributes": {
-		"imageAlign": {
+		"typographySize": {
 			"type": "string",
-			"default": "center center",
-			"variable": "custom"
+			"default": "120-default"
 		}
 	},
-	"options": {
-		"imageAlign": {
-			"top left": {
-				"horizontal": "flex-start",
-				"vertical": "flex-start"
-			},
-			"top center": {
-				"horizontal": "center",
-				"vertical": "flex-start"
-			},
-			"top right": {
-				"horizontal": "flex-end",
-				"vertical": "flex-start"
-			},
-			"center left": {
-				"horizontal": "flex-start",
-				"vertical": "center"
-			},
-			"center center": {
-				"horizontal": "center",
-				"vertical": "center"
-			},
-			"center right": {
-				"horizontal": "flex-end",
-				"vertical": "center"
-			},
-			"bottom left": {
-				"horizontal": "flex-start",
-				"vertical": "flex-end"
-			},
-			"bottom center": {
-				"horizontal": "center",
-				"vertical": "flex-end"
+	"variables": {
+		"typographySize": [
+			{
+				"variable": {
+					"typography-size": "120px"
+				}
 			},
-			"bottom right": {
-				"horizontal": "flex-end",
-				"vertical": "flex-end"
+			{
+				"breakpoint": "tablet",
+				"inverse": true,
+				"variable": {
+					"typography-size": "80px"
+				}
 			}
-		}
-	},
+			{
+				"breakpoint": "mobile",
+				"inverse": true,
+				"variable": {
+					"typography-size": "320px"
+				}
+			}
+		]
+	}
 }
 ```
 
 **Output:**
 ```css
---image-align: center center; 
---image-align-horizontal: center;
---image-align-vertical: center;
-```
-
-## Manual variables
+.typography[data-id='210c9bbf733ef5c6aa74c49168ac29a7'] {
+	--typography-size: 120px;
+}
 
-There is an option to add custom CSS variables that get output independently from all the attributes. Just add a top-level `variables` key inside the manifest and add each variable as an array item.
-Manual variables will be added at the end of the output.
+@media(max-width: 1279px) {
+	.typography[data-id='210c9bbf733ef5c6aa74c49168ac29a7'] {
+		--typography-size: 80px;
+	}
+}
 
-**Manifest:**
-```json
-{
-	"attributes": {
-		// ...
-	},
-	"variables": [
-		"--variable1: test1",
-		"--variable2: test2",
-		"--variable3: test3"
-	]
+@media(max-width: 479px) {
+	.typography[data-id='210c9bbf733ef5c6aa74c49168ac29a7'] {
+		--typography-size: 30px;
+	}
 }
 ```
 
-**Output:**
-```css
---variable1: test1;
---variable2: test2;
---variable3: test3;
-```
- 
-## Manual variables inside the Block editor
+### Magic Variable
```

Comment:
Should we call this magic? Maybe a `Placeholder Variable` would be a better name?

Reply:
I have no strong opinion about this.

Maybe Attribute Value?

Reply:
@infinum/wordpress-team suggestions?

Reply:
`Attribute value replacement` maybe?

Reply:
i have updated it